packages/lab/src/toast/Toast.css

```diff
@@ -0,0 +1,56 @@
+/* Styles applied to Toast root element */
+.saltToast {
+  --toast-background: var(--salt-container-primary-background);
+
+  background: var(--saltToast-background, var(--toast-background));
+  border-color: var(--saltToast-borderColor, var(--toast-borderColor));
+  border-width: var(--saltToast-borderWidth, var(--salt-size-border));
+  border-style: var(--saltToast-borderStyle, var(--salt-container-borderStyle));
+
+  box-sizing: border-box;
+  box-shadow: var(--salt-overlayable-shadow-popout);
```

Comment:
I feel we could take this work as an opportunity to get the new shadow ramp in (see the sheet) similarly done to https://github.com/jpmorganchase/salt-ds/pull/1939